Automattic/jetpack
# mu-wpcom: add coming-soon code from ETK

#### Changes proposed in this Pull Request:

Migration of coming-soon feature from the ETK plugin.

#### Other information:


#### Jetpack product discussion

PT: p9dueE-6jY-p2

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

- Review the code. We aren't loading `jetpack-mu-wpcom` on sites just yet.
- Merged D94672-code which should allow usage of `bin/jetpack-downloader test jetpack-mu-wpcom add/coming-soon-to-wpcom-mu`
- Calypso check added in https://github.com/Automattic/wp-calypso/pull/71012
  - Merged and deployed to simple/Atomic D94911-code
